src/components/Product.vue

```diff
@@ -3,6 +3,8 @@
     <h1>{{ currentProduct }}</h1>
     <h2>Price: ${{ price.toFixed(2) }}</h2>
     <img :src="img">
+    <br>
+    Summary: {{ summary }}
     <h4>
```

Comment:
Semantically, its not compliant to put a `<select>` inside an `<h>` and I think this would not pass validation. The text "Choose a Color" should be in a `<label>` that is associated with the `<select>`

```
<label for="colorSelect">Select a Color</label>
<select id="colorSelect" name="colorSelect">
    <option>
    <option>
</select>
````